aspnetcore/mvc/views/overview.md

```diff
@@ -1,228 +1,336 @@
 
-## Benefits of Using Views
+[View components](xref:mvc/views/view-components) are similar to partial views in that they allow you to reduce repetitive code, but they're appropriate for view content that requires code to run on the server in order to render the webpage. View components are useful when the rendered content requires database interaction, such as for a website shopping cart. View components aren't limited to model binding in order to produce webpage output.
 
-Views provide [separation of concerns](http://deviq.com/separation-of-concerns/) within an MVC app, encapsulating user interface level markup separately from business logic. ASP.NET MVC views use [Razor syntax](razor.md) to make switching between HTML markup and server side logic painless. Common, repetitive aspects of the app's user interface can easily be reused between views using [layout and shared directives](layout.md) or [partial views](partial.md).
+## Benefits of using views
 
-## Creating a View
+Views help to establish a [**S**eparation **o**f **C**oncerns (SoC) design](http://deviq.com/separation-of-concerns/) within an MVC app by separating the user interface markup from other parts of the app. Following SoC design makes your app modular, which provides several benefits:
 
-Views that are specific to a controller are created in the *Views/[ControllerName]* folder. Views that are shared among controllers are placed in the */Views/Shared* folder. Name the view file the same as its associated controller action, and add the *.cshtml* file extension. For example, to create a view for the *About* action on the *Home* controller, you would create the *About.cshtml* file in the */Views/Home* folder.
+* The app is easier to maintain because it's better organized. Views are generally grouped by app feature. This makes it easier to find related views when working on a feature.
+* The parts of the app aren't tightly coupled. You can build and update the app's views separately from the business logic and data access components. You can modify the views of the app without necessarily having to update other parts of the app.
+* It's easier to test the user interface parts of the app because the views are separate units.
+* Due to better organization, it's less likely that you'll accidently repeat sections of the user interface.
 
-A sample view file (*About.cshtml*):
+## Creating a view
 
-[!code-html[Main](../../common/samples/WebApplication1/Views/Home/About.cshtml)]
+Views that are specific to a controller are created in the *Views/[ControllerName]* folder. Views that are shared among controllers are placed in the *Views/Shared* folder. To create a view, add a new file and give it the same name as its associated controller action with the *.cshtml* file extension. To create a view for the *About* action in the *Home* controller, create an *About.cshtml* file in the *Views/Home* folder:
```

Comment:
"To create a view for the About action in the Home controller, create an About.cshtml file in the Views/Home folder" >
"To create a view that corresponds with About action in the Home controller, create an About.cshtml file in the Views/Home folder"